patchwerk-sim.py

```diff
@@ -145,58 +172,74 @@ def run_simulation(tanks_list):
     PATCHWERK = 0
     event_heap = []
     heapq.heappush(event_heap, Event(PATCHWERK, 0))
-    # print("Patchwerk first Hateful Strike scheduled to land at 0 seconds")
-    for ii in range(1, 10):
-        _, _, cast_time = get_heal('h4')
-        start = round(random.random() * cast_time, 1)
-        # print("Healer #{} randomly scheduled to land first heal at {} seconds".format(ii, start))
-        heapq.heappush(event_heap, Event(ii, start))
-
-    # for analysis
-    total_raw_healing = 0
-    total_overhealing = 0
-
-    elapsed = 0
-    heapq.heapify(event_heap)
-    while True:
-        next_event = heapq.heappop(event_heap)
-        elapsed = next_event._time
-        if elapsed >= FIGHT_LENGTH:
-            break
-        # print("{} {}".format(tanks_health, next_event))
-        if next_event.is_hateful():
-            tank, target_idx = get_hateful_target(tanks_list)
-            death, dmg = tank.get_smashed()
-            damage_taken[target_idx] += dmg
-            amount_of_hateful_strikes[target_idx] += 1
-
-            if death:
+    with open('logs.txt', 'a') as f:
```

Comment:
note to ourselves: we can append timestamps to log file names so they don't overwrite themselves when we run this repeatedly